apps/extension/src/service-worker.ts

```diff
@@ -126,14 +95,13 @@ const getServiceHandler = async () => {
       contextValues.set(stakingClientCtx, stakingClient);
       contextValues.set(servicesCtx, services);
       contextValues.set(approverCtx, approveTransaction);
-      contextValues.set(fvkCtx, FullViewingKey.fromJsonString(wallet0.fullViewingKey));
+      contextValues.set(fvkCtx, getFullViewingKey);
 
       return Promise.resolve({ ...req, contextValues });
     },
   });
 };
 
 await fixEmptyGrpcEndpointAfterOnboarding();
-await waitUntilGrpcEndpointExists();
-const handler = await getServiceHandler();
+const handler = getServiceHandler(await onboardGrpcEndpoint());
```

Comment:
Maybe it's my IDE, but it seems like `await onboardGrpcEndpoint()` is returning a `never` type